andela/denethor-ah-backend
# 163478796 Users should be able to reset password reset via email

> #### What does this PR do?
> * Implement password reset via email
> 
> #### Description of Task to be completed?
> * Create forgot password and change password routes
> * Setup Controller class to implement feature
> * Install the required package dependencies
> 
> #### How should this be manually tested?
> * Clone the repo `git clone https://github.com/andela/denethor-ah-backend.git`
> * Install Postgresql from [here](https://www.postgresql.org/download/)
> * Create a database
> * Checkout to the branch `git checkout -b feature/163478796/Reset-Password`
> * Install application dependencies `npm install`
> * Run the tests `npm test`
> 

> #### What are the relevant pivotal tracker stories?
> [Users should be able to reset password reset via email](https://www.pivotaltracker.com/story/show/163478796)
